### PR TITLE
Task/AMP-24046/charts

### PR DESCRIPTION
**Pull request by ginchauspe**

### CODE REVIEW

File: amp/TEMPLATE/ampTemplate/dashboard/build/index.html [hunk 1]
```diff
@@ -26,26 +25,8 @@ <h2><small>Loading...</small></h2>
     </div>
  	<div id="amp-footer">
     </div>
-    <script src="/TEMPLATE/ampTemplate/js_2/translations/common.js"></script>
     <script src="/TEMPLATE/ampTemplate/commonJs/common.min.js"></script>
-
     <script src="compiled-js/app.js"></script>
-    <script language="JavaScript">
```
Comment (jdeanquin-dg) [inline amp/TEMPLATE/ampTemplate/dashboard/build/index.html]: why did you remove this code?

File: amp/TEMPLATE/ampTemplate/dashboard/dev/app/js/app/models/enabled-charts-collection.js [hunk 2]
```diff
@@ -1,38 +1,40 @@
...
+                'Cache-Control': 'no-cache'
+            },
+            data: JSON.stringify(params), // This is necessary due to
+            // incompatibilities with Jersey
+            // when receiving the params.
+            error: function (collection, response) {
+                console.error('error loading charts.');
+            },
+            success: function (collection, response) {
+                // console.log(response);
```
Comment (jdeanquin-dg) [inline amp/TEMPLATE/ampTemplate/dashboard/dev/app/js/app/models/enabled-charts-collection.js]: can we removed this log?

File: amp/TEMPLATE/ampTemplate/dashboard/dev/app/js/app/models/enabled-charts-collection.js [hunk 3]
```diff
@@ -1,38 +1,40 @@
...
+            traditional: true,
+            headers: {
+                'Content-Type': 'application/json',
+                'Cache-Control': 'no-cache'
+            },
+            data: JSON.stringify(params), // This is necessary due to
+            // incompatibilities with Jersey
+            // when receiving the params.
+            error: function (collection, response) {
+                console.error('error loading charts.');
```
Comment (jdeanquin-dg) [inline amp/TEMPLATE/ampTemplate/dashboard/dev/app/js/app/models/enabled-charts-collection.js]: can we report the error to the user?

File: amp/TEMPLATE/ampTemplate/dashboard/dev/app/js/app/views/main.js [hunk 4]
```diff
@@ -14,139 +14,190 @@ var PredictabilityChart = require('../models/chart-aid-predictability');
...
+        heatmapsConfigs.fetchData();
+
+        if (enabledChartsFM.models[0].get('error') !== undefined) {
+            // The same endpoint will send an error if 'DASHBOARDS' is not active in the Feature Manager.
+            window.location = '/';
+        }
+
+        var col = [];
+        if (_.find(enabledChartsFM.models[0].get('DASHBOARDS'), function (item) {
+            return item.indexOf('/Dashboards[true]/Top Donors[true]') > -1;
```
Comment (jdeanquin-dg) [inline amp/TEMPLATE/ampTemplate/dashboard/dev/app/js/app/views/main.js]: can we use constants for literals?

File: amp/TEMPLATE/ampTemplate/dashboard/dev/app/js/app/models/enabled-charts-collection.js (hunk 3 above)
Comment (ginchauspe) [reply amp/TEMPLATE/ampTemplate/dashboard/dev/app/js/app/models/enabled-charts-collection.js]: No, this function shouldnt throw errors and it would take time to implement.
